libass/ass.h

```diff
@@ -571,6 +579,15 @@ ASS_Track *ass_new_track(ASS_Library *);
  */
 int ass_track_set_feature(ASS_Track *track, ASS_Feature feature, int enable);
 
+/**
+ * \brief Check if a feature is enabled
+ * \param track track
+ * \param feature the specific feature to check the state of
+ * \return 0 if disabled, a positive integer with feature-dependent meaning if
+ * enabled, -1 if feature is unknown
```

Comment:
"[P]ositive integer" left intentionally vague since I remember you folks mentioned you may want to have non-boolean states in future.

Reply:
Also should be mentioned behavior in case of macro features like `ASS_FEATURE_INCOMPATIBLE_EXTENSIONS`.